src/app/cube/details/details.component.ts

```diff
@@ -230,6 +229,21 @@ export class DetailsComponent implements OnInit, OnDestroy {
       this.revisedChildren.length);
   }
 
+  /**
+   * Gets if the relevancy banner should be displayed if the next
+   * check date is within a month before/after the date
+   */
+  get relevancyBannerStatus(): boolean {
```

Comment:
I would only display 1 month before relevancy and disappear only after relevancy has been updated